src/lib/models/event.js

```diff
@@ -106,7 +106,12 @@ export default class Event {
   }
 
   get httpServerResponse() {
-    return this.returnEvent.http_server_response;
+    const response = { ...this.returnEvent.http_server_response };
+    if (response.status_code) {
+      response.status = response.status_code;
+      delete response.status_code;
+    }
+    return response;
```

Comment:
I moved this to occur during normalization as I think it makes the most sense. I also wanted to avoid the allocation on line 109. I'm not sure if it would break anything in this case, but we want to maintain stable references where ever we can.